### PR TITLE
Added script for granting permissions

### PR DESCRIPTION
**Pull request by xiaorongyang**

### CODE REVIEW

**Review by SathishMeldcx — CHANGES_REQUESTED** [review]
I would suggest adding AgentM related permission as well which includes both Portrait and Landscape.

File: grant-permissions.bat [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@echo off
+adb version
+echo MeldCX note: ADB Version must be 1.0.33 or above
+
+adb root
+timeout /t 20 /NOBREAK
```
Comment (SathishMeldcx) [inline grant-permissions.bat]: After adb root, do we need 20 seconds to sleep? I feel 1 or 2 seconds is enough

File: grant-permissions.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+command -v adb >/dev/null 2>&1 || { echo >&2 "I require adb but it's not installed.  Aborting."; exit 1; }
+adb root
+sleep 10s
```
Comment (SathishMeldcx) [inline grant-permissions.sh]: You can reduce the sleep here as well.

**Review by SathishMeldcx — APPROVED** [review]

File: grant-permissions.bat (hunk 1 above)
Comment (xiaorongyang) [reply grant-permissions.bat]: Cool, I will reduce to 5s then, I notice it took more than 3s on my side sometimes.

**Pull request merged**